cmd/pulumi.go

```diff
@@ -226,6 +229,14 @@ func getCLIVersionInfo() (semver.Version, semver.Version, error) {
 		return latest, oldest, err
 	}
 
+	brewLatest, isBrew, err := getLatestBrewFormulaVersion()
+	if err != nil {
+		return semver.Version{}, semver.Version{}, err
+	}
+	if isBrew {
+		return brewLatest, brewLatest, nil
```

Comment:
Do we want to cache this information like we do when requesting it from our service?

Reply:
Yep--just fixed that :)